addons/survey/models/survey_question.py

```diff
@@ -444,6 +444,8 @@ def _get_stats_summary_data(self, user_input_lines):
             stats.update(self._get_stats_summary_simple(user_input_lines))
             if self.question_type == 'numerical_box':
                 stats.update(self._get_stats_summary_data_numerical(user_input_lines))
+            if self.question_type == 'date':
+                stats.update(self._get_stats_summary_common_lines(user_input_lines))
```

Comment:
Same here, don't extract to a method, do it inline.

Reply:
Done